### PR TITLE
OCPBUGS-20097: Migrate tools image to RHEL9

### PR DESCRIPTION
**Pull request by soltysh**
/assign @ardaguclu

### CODE REVIEW

**Comment by ardaguclu** [comment]
/lgtm
/hold
Until CI is green

File: images/tests/Dockerfile.rhel [hunk 1]
```diff
@@ -12,7 +12,7 @@ COPY --from=builder /go/src/github.com/openshift/origin/zz_generated.manifests/*
 
 RUN PACKAGES="git gzip util-linux" && \
     if [ $HOSTTYPE = x86_64 ]; then PACKAGES="$PACKAGES python3-cinderclient"; fi && \
-    if [ $HOSTTYPE = x86_64 ]; then PACKAGES="$PACKAGES rt-tests rteval"; fi && \
+    if [ $HOSTTYPE = x86_64 ]; then PACKAGES="$PACKAGES realtime-tests rteval"; fi && \
```
Comment (dinhxuanvu) [inline images/tests/Dockerfile.rhel:15]: The CI reports `No match for argument: realtime-tests`. So this rpm is not available?

Comment (soltysh) [reply images/tests/Dockerfile.rhel:15]: Wrong base image is being used, it's rhel8, and should be rhel9

**Comment by soltysh** [comment]
With https://github.com/openshift/release/pull/51253 in
/retest

**Comment by ardaguclu** [comment]
I think, we have a sufficient signal to merge this.

**Pull request merged**